#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Keep no_NO around during transition to nb_NO
# DP: Related bugs: 
# DP: Dpatch author:  Jeff Bailey
# DP: Patch author: Petter Reinholdtsen
# DP: Upstream status: Debian-Specific
# DP: Status Details: 
# DP: Date: Wed Mar 24 10:35:44 EST 2004, (Updated 2005-01-02 gotom)

PATCHLEVEL=0

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p$PATCHLEVEL < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p$PATCHLEVEL < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
2005-01-02  GOTO Masanori  <gotom@debian.org>

	* intl/locale.alias: Dropped.

--- /dev/null
+++ localedata/locales/no_NO
@@ -0,0 +1,93 @@
+escape_char	/
+comment_char    %
+
+% This is NOT the Norwegian language locale for Norway
+% You really want nb_NO instead.
+% Source: Norsk Standardiseringsforbund
+% Address: University Library,
+%   Drammensveien 41, N-9242 Oslo, Norge
+% Contact: Kolbjoern Aamboe
+% Tel: +47 - 22859109
+% Fax: +47 - 22434497
+% Email: kolbjorn.aambo@usit.uio.no
+% Language: no
+% Territory: NO
+% Revision: 4.3
+% Date: 1996-10-15
+% Application: general
+% Users: general
+% Repertoiremap: mnemonic.ds
+% Charset: ISO-8859-1
+% Distribution and use is free, also
+% for commercial purposes.
+
+LC_IDENTIFICATION
+title      "Norwegian (Bokmal) locale for Norway"
+source     "Norsk Standardiseringsforbund"
+address    "University Library,, Drammensveien 41, N-9242 Oslo, Norge"
+contact    ""
+email      "bug-glibc@gnu.org"
+tel        ""
+fax        ""
+language   "Norwegian Bokm<U00E5>l"
+territory  "Norway"
+revision   "1.0"
+date       "2000-06-29"
+%
+category  "nb_NO:2000";LC_IDENTIFICATION
+category  "nb_NO:2000";LC_CTYPE
+category  "nb_NO:2000";LC_COLLATE
+category  "nb_NO:2000";LC_TIME
+category  "nb_NO:2000";LC_NUMERIC
+category  "nb_NO:2000";LC_MONETARY
+category  "nb_NO:2000";LC_MESSAGES
+category  "nb_NO:2000";LC_PAPER
+category  "nb_NO:2000";LC_NAME
+category  "nb_NO:2000";LC_ADDRESS
+category  "nb_NO:2000";LC_TELEPHONE
+
+END LC_IDENTIFICATION
+
+LC_COLLATE
+copy "nb_NO"
+END LC_COLLATE
+
+LC_CTYPE
+copy "nb_NO"
+END LC_CTYPE
+
+LC_MONETARY
+copy "nb_NO"
+END LC_MONETARY
+
+LC_NUMERIC
+copy "nb_NO"
+END LC_NUMERIC
+
+LC_TIME
+copy "nb_NO"
+END LC_TIME
+
+LC_MESSAGES
+copy "nb_NO"
+END LC_MESSAGES
+
+LC_PAPER
+copy "nb_NO"
+END LC_PAPER
+
+LC_TELEPHONE
+copy "nb_NO"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+copy "nb_NO"
+END LC_MEASUREMENT
+
+LC_NAME
+copy "nb_NO"
+END LC_NAME
+
+LC_ADDRESS
+copy "nb_NO"
+END LC_ADDRESS
